cmd/gatekeeper-test-alpha/gatekeeper-test-alpha.go

```diff
@@ -1,16 +1,54 @@
 package main
 
 import (
+	"errors"
 	"fmt"
 	"os"
 
+	"github.com/open-policy-agent/gatekeeper/pkg/gktest"
 	"github.com/spf13/cobra"
 )
 
+var run string
+
+func init() {
+	rootCmd.Flags().StringVarP(&run, "run", "r", "",
+		`regular expression which filters tests to run by name`)
+}
+
 var rootCmd = &cobra.Command{
 	Use:   "gatekeeper-test-alpha path [--run=name]",
 	Short: "Gatekeeper Test Alpha is a unit test CLI for Gatekeeper Constraints",
+	Example: `  gatekeeper-test-alpha label-tests.yaml
```

Comment:
Most indentation is automatic, but for `Example` it isn't. I like the two-space indent rather than the default no-indent for examples since it keeps things consistent with the other sections.